test/e2e/apimachinery/aggregator.go

```diff
@@ -56,7 +56,11 @@ const (
 var _ = SIGDescribe("Aggregator", func() {
 	var ns string
 	var c clientset.Interface
-	var aggrclient *aggregatorclient.Clientset
+
+	config, err := framework.LoadConfig()
+	framework.ExpectNoError(err)
```

Comment:
If this is part of setup should it be `framework.Fail` here?

Reply:
same for creating the aggregator client below

Reply:
i guess, yes. will update.